cmd/kubelet/app/options/options.go

```diff
@@ -55,7 +55,7 @@ const defaultRootDir = "/var/lib/kubelet"
 // In general, please try to avoid adding flags or configuration fields,
 // we already have a confusingly large amount of them.
 type KubeletFlags struct {
-	TenantPartitionApiservers []string
+	TenantPartitionKubeConfig string
```

Comment:
will this still be the comma separated strings for multiple kubeconfig files? if, so, using plural might be more clear.

Reply:
Yes, currently it take comma separated strings. Since in scheduler and KCM, all kubeconfig for RP/TP are singular, keep it consistent.